crates/sui-core/src/traffic_controller/mod.rs

```diff
@@ -35,6 +39,22 @@ struct Blocklists {
     proxy_ips: BlocklistT,
 }
 
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub enum KillswitchState {
+    On,
+    Off,
+}
+
+impl From<String> for KillswitchState {
+    fn from(s: String) -> Self {
+        match s.as_str() {
```

Comment:
can you strip whitespace (especially newlines) so that people can easily write to the killswitch file manually?